### PR TITLE
Add possibility to remove timestamp in backup_name

### PR DESCRIPTION
**Pull request by webalexeu**
Hello,

PR proposal to be able to disable timestamp in backup name by setting BACKUP_NAME_TIMESTAMP = false

This is to be able to use versionning in S3 lifecycle policy

Thank you,
Alex

### CODE REVIEW

**Comment by webalexeu** [comment]
Hello @peterrus ,

What do you think about that new feature?

Thanks for your feedback

File: README.md [hunk 1]
```diff
@@ -20,7 +20,8 @@ The container is configured via a set of required environment variables:
 - `S3_BUCKET_URL`: in most cases this should be `s3://name-of-your-bucket/`
 - `AWS_DEFAULT_REGION`: The AWS region your bucket resides in
 - `CRON_SCHEDULE`: Check out [crontab.guru](https://crontab.guru/) for some examples:
-- `BACKUP_NAME`: A name to identify your backup among the other files in your bucket, it will be postfixed with the current timestamp (date and time)
+- `BACKUP_NAME`: A name to identify your backup among the other files in your bucket 
+- `BACKUP_NAME_TIMESTAMP`: It will postfixed `BACKUP_NAME` with the current timestamp (date and time) (Optional, defaults to true)
```
Comment (peterrus) [inline README.md]: Maybe rephrase this into "`BACKUP_NAME` will be postfixed with the current timestamp (etc)" ?

**Review by peterrus — CHANGES_REQUESTED** [review]
Small nitpick but otherwise good to go, thanks for your contribution @webalexeu !

File: README.md (hunk 1 above)
Comment (webalexeu) [reply README.md]: Readme updated as suggested
Thank you
